website/src/theme/DocPage/index.js

```diff
@@ -26,6 +26,26 @@ function DocPageContent({ currentDocRoute, versionMetadata, children }) {
     docsSidebars,
     version,
   } = versionMetadata;
+
+  useEffect(() => {
+    if(docsSidebars[sidebarName][0].label === "General"){
+      document.querySelectorAll(".navbar__link--active")[0].text = "General";
+    } else if (document.getElementById("getting-started")) {
+      document.querySelectorAll(".navbar__link--active")[0].text = "Apache APISIX";
+    } else if (document.getElementById("dashboard")) {
+      document.querySelectorAll(".navbar__link--active")[0].text = "Apache APISIX Dashboard";
+    } else if (document.getElementById("what-is-apisix-ingress-controller")) {
```

Comment:
Maybe we could maintain a mapper here, like:

```
{
  "dashboard": "Apache APISIX Dashboard"
}
```

Reply:
Yes we can, 
Do you also want me to separate the selectors?

Reply:
yes, make it more clearly